.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: purescript-contrib/setup-purescript@main
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "10"
```

Comment:
Is there a specific reason for this version and not `"12"` like for the core libraries?

Reply:
The core libraries were updated before our discussion about the minimum supported node version (10), which will switch to 12 in March. We don’t use any features that are different between the versions so we could use either of these two; I opted to update libraries going forward to use v10 but to leave the libraries already on v12 as they are.

Reply:
Oh I thought the discussion was only for the purescript-node libraries. I’m relieved this is not something to change on the whole org then 😅

Reply:
Yea me too 😅 we’ll just be sure to have those on 10, and in general we should try to use 10, and in the future we will have to update to 12. Unless there’s an easy way with setup-node to automatically target the last supported version.